Formula/jenkins.rb

```diff
@@ -1,8 +1,8 @@
 class Jenkins < Formula
   desc "Extendable open source continuous integration server"
   homepage "https://jenkins-ci.org"
-  url "http://mirrors.jenkins-ci.org/war/1.655/jenkins.war"
-  sha256 "0cee889af697c115961ce50229cc5e39d1b798c0a0a689687b745c0a938c8547"
+  url "http://mirrors.jenkins-ci.org/war/1.656/jenkins.war"
+  sha256 "97085359725b16aca0f419486e5c1809399e33dbe6d906cd55ef82fad76fb30d"
 
   devel do
     url "http://mirrors.jenkins-ci.org/war-rc/2.0/jenkins.war"
```

Comment:
@mikemcquaid I think you're right about that. SHA256 for the jenkins 2.0 war is a moving target. I'll remove the devel version. Should I create a new PR for that?

Also, the test in this formula was modified to make it compatible with 2.0. Should I revert that change too or leave it so that when jenkins 2.0 is eventually released, the test will be good to go?